centrallix/htmlgen/htdrv_tab.c

```diff
@@ -343,7 +349,7 @@ httabRender(pHtSession s, pWgtrNode tree, int z)
 	    htrAddWgtrCtrLinkage_va(s, tabpage_obj, "htr_subel(_parentobj, \"tc%POSpane%POS\")", id, i+1);
 
 	    /** Add DIV section for the tabpage. **/
-	    htrAddBodyItem_va(s,"<div id=\"tc%POSpane%POS\" style=\"POSITION:absolute; VISIBILITY:%STR&CSSVAL; LEFT:0px; TOP:0px; WIDTH:%POSpx; Z-INDEX:%POS;\">\n",
+	    htrAddBodyItem_va(s,"<div id=\"tc%POSpane%POS\" class=\"tcPane\" style=\"POSITION:absolute; VISIBILITY:%STR&CSSVAL; LEFT:0px; TOP:0px; WIDTH:%POSpx; Z-INDEX:%POS;\">\n",
```

Comment:
We should have a CSS rule to to with tcPane.  The static inline styles can be included in that.